external/vulkancts/modules/vulkan/binding_model/vktBindingDescriptorSetRandomTests.cpp

```diff
@@ -2020,9 +2020,9 @@ tcu::TestStatus DescriptorSetRandomTestInstance::iterate (void)
 			VK_SUBPASS_EXTERNAL,							// deUint32				srcSubpass
```

Comment:
this is the SubpassDependency for the the RenderPass. See lines 2232 - 2255 below for where it is used.
Seems like this could also be fixed by adjusting the MemoryBarrier on line 2236?